### PR TITLE
Add more missing JavaDoc class header information

### PR DESCRIPTION
**Pull request by hsaputra**
Another PR to add more missing JavaDoc class header information for public classes.

### CODE REVIEW

**Comment by hsaputra** [comment]
Looks like style check error. Will update PR

**Comment by hsaputra** [comment]
Reopen PR

File: twill-core/src/main/java/org/apache/twill/internal/TwillContainerLauncher.java [hunk 1]
```diff
@@ -68,6 +68,16 @@ public TwillContainerLauncher(RuntimeSpecification runtimeSpec, ProcessLauncher.
     this.secureStoreLocation = secureStoreLocation;
   }
 
+  /**
+   * Start execution run for a class in a container. Will return instance of {@link TwillContainerController}
+   * that allows send messages directly to the container.
```
Comment (anew) [inline twill-core/src/main/java/org/apache/twill/internal/TwillContainerLauncher.java]: send/sending

File: twill-core/src/main/java/org/apache/twill/internal/TwillContainerLauncher.java [hunk 2]
```diff
@@ -68,6 +68,16 @@ public TwillContainerLauncher(RuntimeSpecification runtimeSpec, ProcessLauncher.
...
+   *
+   * @param runId Use to represent unique id of the container run.
+   * @param instanceId The Twill instance Id
+   * @param mainClass The main class will be run in the a container.
```
Comment (anew) [inline twill-core/src/main/java/org/apache/twill/internal/TwillContainerLauncher.java]: The main class to run in the container

Comment (anew) [inline twill-core/src/main/java/org/apache/twill/internal/TwillContainerLauncher.java]: What is correct convention for @param? End it with a . or not? In any case you should be consistent about that

File: twill-yarn/src/main/hadoop21/org/apache/twill/internal/yarn/Hadoop21YarnAMClient.java [hunk 3]
```diff
@@ -40,7 +40,7 @@
 import javax.annotation.Nullable;
 
 /**
- *
+ * Wrapper class for {@link AMRMClient} for Hadoop version 2.1 or greater.*
```
Comment (anew) [inline twill-yarn/src/main/hadoop21/org/apache/twill/internal/yarn/Hadoop21YarnAMClient.java]: remove the * at the end of the line

File: twill-core/src/main/java/org/apache/twill/internal/TwillContainerLauncher.java (hunk 2 above)
Comment (hsaputra) [inline twill-core/src/main/java/org/apache/twill/internal/TwillContainerLauncher.java]: Seemed like most ended with period. Will add to them

**Comment by anew** [comment]
Thanks for fixing Javadocs. I did have a couple of comments, though.

**Comment by hsaputra** [comment]
Thanks for the reviews, just updated the PR to address the comments.

File: twill-core/src/main/java/org/apache/twill/internal/TwillContainerLauncher.java (hunk 2 above)
Comment (anew) [inline twill-core/src/main/java/org/apache/twill/internal/TwillContainerLauncher.java]: Looks like we Java itself does not put a period after the @param. Only if it is a complete sentence. So you would either say:
{code}
@param mainClass The main class to run in the container
{code}
or
{code}
@param mainClass This is the main class that will run in the container.
{code}
I think typically the first one is preferred.

**Comment by anew** [comment]
LGTM

**Comment by chtyim** [comment]
Can you rebase the commits into one?

**Comment by hsaputra** [comment]
Done, thanks

**Pull request closed without merging**